### DISCUSSION TITLE
WARN http proxy: Unsupported scheme

### DISCUSSION THREAD

**Discussion by invinciberry** [qdm12/gluetun#729] [Q&A]
Seeing this in the log. Not sure whats going on.

**Comment by qdm12** [comment]
The HTTP proxy only supports tunneling http and https requests, not other schemes.

Relevant code:

https://github.com/qdm12/gluetun/blob/e7c952cbf78951a688782861dbf9fde6c6bc87e7/internal/httpproxy/http.go#L15

**Reply by invinciberry** [reply]
Thanks for the reply. For some reason I am only getting "ARN http proxy: Unsupported scheme" without the "request.URL.Scheme" part attached. 
Also I am using gluetun as an http proxy on my IOS device with a custom pac pointed to the server. Not sure if thats the issue.

**Reply by qdm12** [reply]
Maybe something tries to access urls without a scheme. I'd suggest you to use shadowsocks which is an encrypted socks5 proxy, they most likely have apps on ios. It also supports tunneling udp which allows you to also tunnel DNS unlike the http proxy.

**Reply by invinciberry** [reply]
Sorry for the confusion, when I say iOS device, what I really mean is my Apple TV. There are no VPN client on it, but I was able to enable http proxy through apple configurator.
